packages/manager/src/features/linodes/LinodesDetail/LinodeNetworking/IPSharing.tsx

```diff
@@ -103,22 +88,77 @@ const IPSharingPanel: React.FC<CombinedProps> = (props) => {
     open,
     onClose,
     linodeSharedIPs,
+    availableRanges,
   } = props;
 
-  const { data, isLoading } = useAllLinodesQuery(
+  const availableRangesMap: AvailableRangesMap = formatAvailableRanges(
+    availableRanges
+  );
+
+  const resp = useAllLinodesQuery(
     { page_size: API_MAX_PAGE_SIZE },
     {
       region: linodeRegion,
     },
     open // Only run the query if the modal is open
   );
 
+  let isLoading = resp.isLoading;
+  const data = resp.data;
```

Comment:
This can be simplified to

```suggestion
  { data: linodesData, isLoading } = useAllLinodesQuery(
    { page_size: API_MAX_PAGE_SIZE },
    {
      region: linodeRegion,
    },
    open // Only run the query if the modal is open
  );
```

Reply:
^ oops, forgot the `const`

Reply:
just remembered I didn't do this because isLoading can't be a const

Reply:
unless theres a way around that